inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp

```diff
@@ -877,4 +906,95 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_1D, GroupConvolutionLayerCPUTest,
+        ::testing::Combine(
+                ::testing::Combine(
+                        groupConvParams_ExplicitPadding_Gemm_2D,
+                        ::testing::Values(Precision::FP32),
+                        ::testing::Values(Precision::U8 /*, Precision::I8*/), // i8 primitives are disabled in oneDNN fork
+                        ::testing::Values(Precision::FP32, Precision::U8, Precision::I8),
+                        ::testing::Values(InferenceEngine::Layout::ANY),
+                        ::testing::Values(InferenceEngine::Layout::ANY),
+                        ::testing::Values(std::vector<size_t >({2, 12, 7, 7})),
+                        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                ::testing::ValuesIn(filterCPUInfoForDevice({conv_gemm_2D_nspc})),
+                ::testing::ValuesIn(fusingParamsSetI8)),
+        GroupConvolutionLayerCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_3D_Gemm_I8, GroupConvolutionLayerCPUTest,
+        ::testing::Combine(
+                ::testing::Combine(
+                        groupConvParams_ExplicitPadding_Gemm_3D,
+                        ::testing::Values(Precision::FP32),
+                        ::testing::Values(Precision::U8 /*, Precision::I8*/), // i8 primitives are disabled in oneDNN fork
+                        ::testing::Values(Precision::FP32, Precision::U8, Precision::I8),
+                        ::testing::Values(InferenceEngine::Layout::ANY),
+                        ::testing::Values(InferenceEngine::Layout::ANY),
+                        ::testing::Values(std::vector<size_t >({2, 12, 7, 7, 7})),
+                        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                ::testing::ValuesIn(filterCPUInfoForDevice({conv_gemm_3D_nspc})),
+                ::testing::ValuesIn(fusingParamsSetI8)),
+        GroupConvolutionLayerCPUTest::getTestCaseName);
+
+const std::vector<CPUSpecificParams> CPUParams_2D_I8 = {
+        conv_sse42_2D_nspc,
+        conv_avx2_2D_nspc,
+        conv_avx512_2D_nspc
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_2D_I8, GroupConvolutionLayerCPUTest,
+                         ::testing::Combine(
+                                 ::testing::Combine(
+                                         groupConvParams_ExplicitPadding_2D,
+                                         ::testing::Values(Precision::FP32),
+                                         ::testing::Values(Precision::U8 /*, Precision::I8*/), // i8 primitives are disabled in oneDNN fork
+                                         ::testing::Values(Precision::FP32, Precision::U8, Precision::I8),
+                                         ::testing::Values(InferenceEngine::Layout::ANY),
+                                         ::testing::Values(InferenceEngine::Layout::ANY),
+                                         ::testing::Values(std::vector<size_t >({2, 64, 7, 7})),
```

Comment:
Here we testing only channels shape evenly divided by the SIMD vector size, that could hide some problems related to tails processing. Suggest adding more shape variations where input and output channels number is a fraction of blocks number. Please check other new tests instances including other layers.